lms/static/scripts/frontend_apps/components/DeepLinkingFilePickerFormFields.tsx

```diff
@@ -0,0 +1,27 @@
+export type DeepLInkingFilePickerFormFieldsProps = {
```

Comment:
This component is actually entirely generic already, so you could call it `HiddenFormFields` and name the props accordingly. In that case the prop name could be just `fields`.

If you decided not to do that, I would at least simplify the name to eg. `DeepLinkingFormFields` and the prop name to just `fields`.

Reply:
> so you could call it HiddenFormFields 

Went for this :+1: